### PR TITLE
Update prerequisites for Windows package

### PR DESCRIPTION
**Pull request by kenhys**
It is helpful not to shoot yourself in the foot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,12 +16,23 @@ td-agent-builder is a new build system for [td-agent](http://docs.treasuredata.c
...
   * [Docker Desktop for Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows)
     * You need to switch to "Windows containers" before using it.
   * [RubyInstaller](https://rubyinstaller.org/) 2.4 or later.
+    * You need to install MSYS2 development toolchain too.
   * [Git for Windows](https://gitforwindows.org/)
 
+After installed above software, you need to enable additional features from powershell (as admin).
+
+```
+dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
```
Comment (cosmo0920) [inline README.md:29]: This feature should be needed for Docker Desktop for Windows to run Linux container, right?
Just for normal Docker Desktop for Windows installation?

File: README.md [hunk 2]
```diff
@@ -16,12 +16,23 @@ td-agent-builder is a new build system for [td-agent](http://docs.treasuredata.c
...
+dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart
```
Comment (cosmo0920) [inline README.md:30]: This is also needed for WSL2?

File: README.md [hunk 3]
```diff
@@ -16,12 +16,23 @@ td-agent-builder is a new build system for [td-agent](http://docs.treasuredata.c
 
 ### For building Windows package (.msi)
 
-  * Windows OS (10 or 2019 are verified)
+  * Windows OS (10 Pro or 2019 are verified)
   * [Docker Desktop for Windows](https://hub.docker.com/editions/community/docker-ce-desktop-windows)
     * You need to switch to "Windows containers" before using it.
   * [RubyInstaller](https://rubyinstaller.org/) 2.4 or later.
+    * You need to install MSYS2 development toolchain too.
```
Comment (ashie) [inline README.md:23]: Is it really required on host OS?

File: README.md (hunk 1 above)
Comment (ashie) [inline README.md:29]: Is it really required (for windows containers)?

File: README.md [hunk 4]
```diff
@@ -16,12 +16,23 @@ td-agent-builder is a new build system for [td-agent](http://docs.treasuredata.c
...
   * [Git for Windows](https://gitforwindows.org/)
 
+After installed above software, you need to enable additional features from powershell (as admin).
+
+```
+dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart
+dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart
+dism.exe /online /enable-feature /featurename:Microsoft-Hyper-V /all /norestart
```
Comment (cosmo0920) [inline README.md:31]: For running Windows containers, we should execute the following commands:

```cmd
dism.exe /online /enable-feature /featurename:Containers /all/ norestart
```

File: README.md (hunk 3 above)
Comment (kenhys) [reply README.md]: Removed.

File: README.md (hunk 1 above)
Comment (kenhys) [reply README.md]: Removed.

Comment (kenhys) [reply README.md]: Removed because it is not needed for windows container.

File: README.md (hunk 2 above)
Comment (kenhys) [reply README.md]: ditto.

**Comment by kenhys** [comment]
I've updated PR to reflect feedback.

**Comment by ashie** [comment]
Thanks!

**Pull request merged**